jmx-scraper/src/main/resources/hbase.yaml

```
@@ -7,6 +7,7 @@ rules:
     unit: "{server}"
     type: updowncounter
     mapping:
+      # Group of properties to build hbase.master.region_server.count metric
```

```suggestion
      # hbase.master.region_server.count
```
[nitpicking] Maybe simplify this even further as everything here is a metric (and do the same with others that match `Group of properties to build .* metric` regex)

---

Done